### DISCUSSION TITLE
UX improvement: Right-click in asset-library

### DISCUSSION THREAD

**Discussion by jonasdenil** [craftcms/cms#8675] [Ideas]
### Description
When managing assets in the assets-library, you can easily add a new subfolder by using right-click. But when you want to replace an image, rename a file, etc; you have to select the image, head over to the cog-icon and select your action over there. 
![Cog-menu](https://i.imgur.com/p4nIBqw.png)

Seems like a small issue but it can be time-consuming and confusing for a content-manager or client. Mainly because Craft sometimes gives you a "Craft-right-click-menu" and sometimes you get a system right-click-menu. 
![System right-click-menu](https://i.imgur.com/6xw3Y03.png)

I think we can solve this pretty easily by adding the same menu-items behind the cog-menu, to a right-click on the asset. 

Let me know your opinion on this. 

_Side-note: No regular GitHub user. Also no dev, just your average UX-designer._

### Additional info

- Craft version: 3.1.25

**Comment by tim-van-dijkhuizen** [comment]
I came here to suggest the same exact thing. Right-clicking should show a menu with all single element actions.

This feature feels like a must because it can save so much time. I recently had too download some assets from a production site and I didn't have FTP access, so I had to download them through Craft. It really hurt how many clicks it took me to download those assets.

A huge thumbs up from me. Really hope this gets implemented soon. This would have been a nice addition to 3.4 😉.
@brandonkelly :)
